src/binance.mjs

```diff
@@ -153,6 +171,10 @@ export class BinanceAccess {
 
         console.log(config);
 
+        return this.send(cmd, config);
+    }
+
+    async send(cmd, config={}) {
```

Comment:
We shouldn't break this out into another function because:
1. it means the API call weight is not tracked if it doesn't come via apiCommand
2. Any error handling added to apiCommand in future will not be used

Reply:
I more intended send() to be private. Unforunately ES6 has no private. Suggest rename to _send() instead?

This is a better way of doing the sends I think so if we do replace axios(as discussed) we can replace it in just 1 spot if that we need to offer more functions which send data other than apiCommand.

Reply:
I'd really rather keep all the communication stuff within the apiCommand function for now

Reply:
Up to you but https://en.wikipedia.org/wiki/Separation_of_concerns.

Reply:
Is that some sort of joke 😂